src/main/java/net/huskycraft/blockyarena/BlockyArena.java

```diff
@@ -122,15 +148,48 @@ private void createDirectories() {
             }
         }
     }
+    
+    private void HandleConfiguration()
+    {
+    	
+    	loader = HoconConfigurationLoader.builder().setPath(defaultConfig).build();
+    	
+         this.rootNode = loader.createEmptyNode(ConfigurationOptions.defaults());
+    	 rootNode.getNode("timers", "lobby", "cooldownSec").setValue(15);
```

Comment:
Everytime the server inits, this code gets called and thus override the existing value in config to 15. Before setting the default to 15, you may want to check if default config already exists.

Reply:
Fixed as of new configuration manager !